.drone.yml

```diff
@@ -33,12 +33,51 @@ steps:
       DIRECTUS_TOKEN:
         from_secret: directus_token
     commands:
+      - export NODE_ENV=staging
+      - npm run export:production
+    when:
+      branch:
+        exclude:
+          - master
+
+  - name: deploy staging
+    image: cupcakearmy/drone-deploy
+    pull: always
+    environment:
+      PLUGIN_KEY:
+        from_secret: key
+    settings:
+      host: love-foundation.org
+      user: root
+      port: 1312
+      target: /srv/staging
+      sources:
+        - ./__sapper__/export
+        - ./docker-compose.staging.yml
+      commands:
+        - docker-compose -f docker-compose.staging.yml up -d --build
+      when:
+        branch:
+          exclude:
+            - master
```

Comment:
wollen wir bei jedem deploy oder vielleicht nur der dev branch? so würde ja bei jedem commit es deployen was nicht master ist. also auch ein feature branch z.b.

Reply:
I think that for now with the little amount of devs working on this project it shouldn't really matter if we deploy each push that isn't on master :)